src/components/sponsor/sponsorBanner.module.css

```diff
@@ -0,0 +1,136 @@
+.hero {
+  color: white;
+  background-image: linear-gradient(to left, rgba(0, 0, 0, 0) 20%, rgba(0, 0, 0, 0.6) 50%, rgba(0, 0, 0, 0) 80%),
+    url(../../assets/images/sponsor/DSC05963.jpg);
+  background-position: left;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: relative;
+  min-height: 465px;
+  height: 95vh;
+}
+
+.heroContainer {
+  margin: 0 auto;
+  max-width: 500px;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.heroTextArea {
+  margin-right: 0px;
+  margin-bottom: 100px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.heroBtnContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.heroBtn {
```

Comment:
A few notes on the buttons:

* The *Get In Touch* button isn't coming through as a different colour for me, so the `.heroBtnVariant` class might be getting overridden somewhere.
* The buttons generally are a bit out of sync with other pages - the index page buttons have more padding and no icons; the previous shows buttons are smaller and all highlighted in the primary colour.
* The hover-over colours are a bit too contrasty for my personal taste, I usually keep the highlight effect quite subtle.

The proper way to resolve this is to write actual `Button` components that we can then reuse throughout the entire website with consistency - another thing for the summer list!